### PR TITLE
[Maxwell]-pros-and-cons-of-using-celo-to-create-decentralized-applications

### PR DESCRIPTION
**Pull request by maxzysparks**
### Thanks for submitting this tutorial for Celo Sage

Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review -

- [x] My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
- [x] The title of the Pull Request matches the title of my tutorial.
- [x] I've followed the tutorial template structure.
- [x] I've proofread the tutorial for spelling and grammar mistakes.
- [x] I've thoroughly explained the code, with inline code comments and text explanations.
- [x] I've run a linter & prettifier to format my code blocks.
- [x] I've run the tutorial to test it and the code works.
- [x] I've followed all the best practices mentioned in the [here](https://docs.celo.org/community/celo-sage/complete-your-tutorial#best-practices).

> NOTE: We cannot review your Pull Request if it does not meet the standard outlined above.
> Reviewers - @joenyzio & @viral-sangani

### CODE REVIEW

File: blog/01-18-2023-pros-and-cons-of-celo-to-create-decentralized-applications [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+authors:
+  - name: Maxwell Onyeka
+    title: Technical writer
+    image_url: https://avatars.githubusercontent.com/u/80678178?v=4
+tags: [celosage, beginner, foundation]
+hide_table_of_contents: true
+slug: /tutorials/pros-and-cons-of-using-celo-to-create-decentralized-applications
+---
+
+![header](../../src/data-tutorials/showcase/beginner/pros.png)
```
Comment (Brisa-Codes) [inline blog/01-18-2023-pros-and-cons-of-celo-to-create-decentralized-applications:13]: update your directory (eliminate "../..")```suggestion
header
```

File: blog/01-18-2023-pros-and-cons-of-celo-to-create-decentralized-applications [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+title: Pros and Cons of using Celo to create Decentralized Applications
+description: Pros and Cons of using Celo to create Decentralized Applications
+authors:
+  - name: Maxwell Onyeka
+    title: Technical writer
+    image_url: https://avatars.githubusercontent.com/u/80678178?v=4
+tags: [celosage, beginner, foundation]
+hide_table_of_contents: true
+slug: /tutorials/pros-and-cons-of-using-celo-to-create-decentralized-applications
+---
```
Comment (Brisa-Codes) [inline blog/01-18-2023-pros-and-cons-of-celo-to-create-decentralized-applications:11]: format the headers with Prettier

Comment (maxzysparks) [reply blog/01-18-2023-pros-and-cons-of-celo-to-create-decentralized-applications]: i've done that

File: blog/01-18-2023-pros-and-cons-of-celo-to-create-decentralized-applications (hunk 1 above)
Comment (maxzysparks) [reply blog/01-18-2023-pros-and-cons-of-celo-to-create-decentralized-applications]: It will affect the image

**Review by viral-sangani — APPROVED** [review]

**Pull request merged**